packages/enzyme-adapter-react-16/.eslintrc

```diff
@@ -1,5 +1,6 @@
 {
   "extends": "airbnb",
+  "parser": "babel-eslint",
```

Comment:
this is also not acceptable; babel-eslint enables all syntax currently, so airbnb projects only use the default parser.
```suggestion
```